src/common.php

```diff
@@ -96,11 +96,17 @@ function isCSRF(&$session, $csrftoken) {
  * @param username The Scratch username whose profile is being linked to
  * @return string A simple <a> link to the user's profile with the contents also being the username
  */
-function linkToScratchProfile(string $username) : string {
+function linkToScratchProfile(string $username, bool $scrollToComments) : string {
```

Comment:
This parameter is not used by anything.

Reply:
yeah, that was what i realised *after* making the pr, but i'm not sure where this function is called when the page opens in a new tab after selecting "submit"... :/